### DISCUSSION TITLE
What happens when you change the license of an archived repository?

### DISCUSSION THREAD

**Discussion by itsmeadarsh2008** [community/community#126044] [Repositories]
### Select Topic Area

General

### Body

Let's say you have a repository called x which is GPLv3 License, I have created a fork which will be MIT License. Is anything wrong going to happen?

**Comment by akathedeveloper** [comment]
Forking a GPLv3-licensed repository and releasing it under the MIT license could create conflicts between licenses. The GPLv3 requires derivative works to be under the same license, while the MIT license is more permissive. This could lead to legal issues and violations of the original license terms. It's important to understand both licenses and ensure compliance to avoid potential problems.
